.github/workflows/weekly.yml

```diff
@@ -35,12 +35,18 @@ jobs:
         }, {
           flink: 1.18-SNAPSHOT,
           branch: main
+        }, {
+          flink: 1.19-SNAPSHOT,
+          branch: main
         }, {
           flink: 1.16.2,
           branch: v3.0
         }, {
           flink: 1.17.1,
           branch: v3.0
+        }, {
+          flink: 1.18.0,
+          branch: v3.0
```

Comment:
Does the v3.0 branch support 1.18.0?

Reply:
I found that you removed the 1.18.0 support in the follow hotfix, and there are compilation failures indeed, so I will 
 also remove the support. thanks for your reminds.
![image](https://github.com/apache/flink-connector-elasticsearch/assets/7313035/ec287fa2-b895-4c70-8694-de327259c304)